### DISCUSSION TITLE
eBus adapter v5 temperature at 64.6° C - normal or too high?

### DISCUSSION THREAD

**Discussion by felix-schwarz** [john30/ebusd#904] [Q&A]
Hi there!

First of all, congratulations to v5 of the eBus adapter! What a terrific piece of hard- and software!

What I'm wondering about is the temperature indicated in the `Pins` section of the web interface. With WiFi turned off and connected via Ethernet (USR-ES1), It reports a temperature of  64.6° C (ambient / room temperature is 25.5° C):

![Bildschirmfoto 2023-05-24 um 12 38 21](https://github.com/john30/ebusd/assets/639669/9b6bcbc7-7bdc-4b36-8f45-5f8cd73f6fca)

Is this a "normal" temperature? Should I add a heatsink?

Thanks!

**Comment by QubaB** [comment]
Temperature of my V5 adapter (WiFi) is 53°C. I think that temperature 63°C is OK (it is below 85°C). See [specification](https://www.espressif.com/en/news/ESP32_C3)

**Comment by felix-schwarz** [comment]
Thanks!

My v5 adapter reported 54° C for me when I used it over WiFi. I switched to Ethernet because I hoped it would reduce the temperature, but that didn't turn out to be the case.

I had a look at the spec sheet prior to posting, but only found the allowed ambient temperature range in it (of which 85° C is the maximum for one ESP32 variant), but unfortunately no maximum temperature for the CPU itself.

I was hoping to get some insight into whether 64°C is a typical, non-problematic temperature for the CPU - or if I should improve its cooling to lengthen its lifetime.

**Reply by QubaB** [reply]
I think that this temperature is pretty OK for such processors. I suggest to ask manufacturer in case of doubts.

**Comment by cweiske** [comment]
My v5 in WiFi mode is 56°C.

**Comment by Dinth** [comment]
My gas boiler is in the loft, where the ambient temperature reaches 60C in summer and ive been using v1+RPi for many years without an issue. I should have v5 in a couple of days so i will do some stress testing
